gitlab/resource_gitlab_project_variable.go

```diff
@@ -155,3 +158,22 @@ func resourceGitlabProjectVariableDelete(d *schema.ResourceData, meta interface{
 	_, err := client.ProjectVariables.RemoveVariable(project, key)
 	return err
 }
+
+func resourceGitlabProjectVariableAugmentClientError(d *schema.ResourceData, err error) error {
```

Comment:
Do we need to prefix all of the function names like that? 🤔

Reply:
Nope haha. Renamed in latest push.